src/w3d/lib/systimer.h

```diff
@@ -54,10 +54,7 @@ inline SysTimeClass::~SysTimeClass()
 }
 
 #ifdef GAME_DLL
-#include "hooker.h"
-
-// This needs to stay here like this to avoid static init issues in cpudetect.
```

Comment:
Not sure what the comment tries to tell us. Did you check that this works correctly?

Reply:
It's a generic comment from years back as to why it is using a make global. At least that's my understanding of it.

Reply:
For reference, this is before I added `#pragma init_seg(lib)` to the setupglobals_zh.cpp file to force the globals to be initialised before other code. Before I did this, the CPUDetect code that runs in the pre-main init code would fail on a call to the timer because the static wasn't initialised in time which isn't an issue when using a standalone build as it wouldn't be a pointer.